### PR TITLE
Add support for specific games

### LINKED ISSUE #14: Add support for specific games
Add game-specific status bars for different games. There are common APIs for popular titles like PUBG, Overwatch, and Rocket League.

### PR DESCRIPTION
**Pull request by oxhacks**
Adds support for PUBG and Overwatch, while also updating branding and framework for displaying specific statistics.

In the future, this will include cleaner integration to switch between games instead of just checking for PUBG.

Resolves #14

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -41,7 +49,7 @@ def get_twitch_stream(user):
     return json.loads(r.text)
 
 
-def build_streamer_json(user, stream, participant_id):
+def build_streamer_json(user, stream, participant_id, user_info):
```
Comment (dbernard) [inline app.py]: Does the `user_info` object also contain the same information we pass in for `user` and `participant_id`? If so, can we do away with `user` and `participant_id` and just pull those details of the `user_info` object?

**Review by dbernard — COMMENTED** [review]
LGTM other than the `user_info` comment. The specific game stats are awesome.

**Comment by oxhacks** [comment]
@dbernard great point RE: redundant parameters. I've updated that function call, and we should be good to go now.

**Comment by dbernard** [comment]
LGTM 👍

**Pull request merged**